docs/content/usage/actions/act-runner.en-us.md

```diff
@@ -114,7 +114,7 @@ If you cannot see the settings page, please make sure that you have the right pe
 
 The format of the registration token is a random string `D0gvfu2iHfUjNqCYVljVyRV14fISpJxxxxxxxxxx`.
 
-A registration token can also be obtained from the gitea [command-line interface](../../administration/command-line.en-us.md#actions-generate-runner-token):
```

Comment:
Seems unnecessary prefix. Just `adminstration/xxx` is okay.

Reply:
![image](https://github.com/go-gitea/gitea/assets/18380374/f573e9d0-1e41-4f4b-a918-03df08e81d84)
You are right. It seems that `../../` is incorrect

Reply:
↑ The log above is incorrect because of the bug in `make prepare-doc` command.
See: https://gitea.com/gitea/gitea-docusaurus/pulls/92

But maybe it is better to let them in a same format.